pkg/apiserver/slowquery/service.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/pingcap/tidb-dashboard/pkg/apiserver/user"
 	"github.com/pingcap/tidb-dashboard/pkg/apiserver/utils"
 	"github.com/pingcap/tidb-dashboard/pkg/tidb"
+	commonUtils "github.com/pingcap/tidb-dashboard/pkg/utils"
 )
```

Comment:
the convension in this project is to name it as utils and alias the one in apiserver as "apiutils" if i remember correctly.